apps/backend/src/modules/permissions/permission.entity.ts

```diff
@@ -0,0 +1,45 @@
+import { type PermissionGetAllItemResponseDto } from "@git-fit/shared";
```

Comment:
You've created a local types.ts for re-exports, you should use it here

Reply:
Good catch! Thanks!